docs/reference/esql/processing-commands/keep.asciidoc

```diff
@@ -40,10 +50,84 @@ include::{esql-specs}/docs.csv-spec[tag=keepWildcard]
 ----
 
 The asterisk wildcard (`*`) by itself translates to all columns that do not
-match the other arguments. This query will first return all columns with a name
+match the other arguments.
+
+This query will first return all columns with a name
 that starts with `h`, followed by all other columns:
 
 [source,esql]
 ----
 include::{esql-specs}/docs.csv-spec[tag=keepDoubleWildcard]
```

Comment:
I think it'd be nice to include the expected results here and in `keepWildcard` above (c.f. https://github.com/elastic/elasticsearch/blob/301bbed5aa4e440c975b5fe9789a48c94cef6f78/x-pack/plugin/esql/qa/testFixtures/src/main/resources/docs.csv-spec#L57-L75)

Reply:
Agree